views/accessibility-statement.html

```diff
@@ -0,0 +1,97 @@
+        <li>Contact us by:</li>
+        <li>email enquiries@companieshouse.gov.uk</li>
+        <li>telephone 0303 1234 500</li>
+        <li>telephone (Welsh language) 029 2038 0065</li>
+        <li>minicom or textphone 029 2022 6788</li>
+      </ul>
+      <p>Or write to us at:</p>
+      <p>Companies House<br>Crown Way<br>Cardiff<br>CF14 3UZ</p>
+      <p>DX 33050</p>
+      <p>We’ll discuss what assistance we can provide to help you meet your responsibilities. With your permission, we’ll place a note on our internal system.</p>
+      <p>For example, if you have impaired vision and would prefer us to contact you by phone, we can call to remind you when your confirmation statement or accounts are due.</p>
+
+      <h2 class="govuk-heading-m">Reporting accessibility problems with this service</h2>
+
+      <p>We are continuing to improve the accessibility of this service. Contact us if you have accessibility requirements that are not listed on this page.</p>
+
+      <h2 class="govuk-heading-m">Enforcement procedure</h2>
+      <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we
+        respond to your complaint,
+        <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS)</a>.</p>
+
+      <h2 class="govuk-heading-m">Technical information about this service’s accessibility</h2>
+      <p>Companies House is committed to making its website accessible in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+      <p>This service is partially compliant with the
+        <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a>
+        AA standard, due to the non-compliances listed below.</p>
+      <p>A page has an illogical heading structure. This fails WCAG 2.4.10 success criterion (section headings). We plan to fix this by April 2020.</p>
+      <p>Some links are out of context. This fails WCAG 2.4.9 success criterion (link purpose). We plan to fix this by April 2020.</p>
+      <p>A links is present which does not indicate it will open in a new window. This fails WCAG 2.4.9 success criterion (link purpose). We plan to fix this by April 2020.</p>
+      <p>A timeout is present without indication to users. This fails WCAG 2.2.6 success criterion (timeouts). We plan to fix this by April 2020.</p>
+
+      <h2 class="govuk-heading-m">How we tested this service</h2>
+      <p>We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA using a variety of manual and automated testing tools.</p>
+      <p>This service was last (externally) audited on 5 November 2018. The audit was carried out by the
```

Comment:
is 2018 the date chs was auditted or extensions? extensions went live 2019.

Reply:
Updated to 2019